### PR TITLE
docs(site): add FAQs section to site

### PR DESCRIPTION
**Pull request by SMQuazi**
**This change:** (check at least one)

- [ ] Adds a new feature
- [ ] Fixes a bug
- [ ] Improves maintainability
- [x] Improves documentation
- [ ] Is a release activity

**Is this a breaking change?** (check one)

- [ ] Yes
- [x] No

**Is the:** (complete all)

- [x] Title of this pull request clear, concise, and indicative of the issue number it addresses, if any?
- [x] Test suite(s) passing?
- [x] Code coverage maximal?
- [x] Changeset added?

**What does this change address?**
There is no FAQs section on the site. 

**How does this change work?**
This change includes a change to the navigation bar and adds a new page that stores content for FAQs.

### CODE REVIEW

File: packages/pharos-site/src/pages/faqs.mdx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  </p>
+</PageSection>
+
+<PageSection
+  title="I tried adopting a component but encountered a number of issues. How do I know when something is usable?"
+  subSectionLevel={2}
+>
+  <p>
+    You can find the status of each component{' '}
+    <PharosLink href="https://pharos.jstor.org/components/component-status">here</PharosLink>. This
```
Comment (daneah) [inline packages/pharos-site/src/pages/faqs.mdx:19]: Should usually use the text of a link to indicate where it's taking you:

```suggestion
    You can find the status of each component on the{' '}
    <PharosLink href="https://pharos.jstor.org/components/component-status">component status page</PharosLink>. This
```

**Review by Niznikr — APPROVED** [review]

File: packages/pharos-site/src/pages/faqs.mdx [hunk 2]
```diff
@@ -0,0 +1,91 @@
+import PageSection from '../../src/components/statics/PageSection.tsx';
+
+<PageSection isHeader title="Pharos FAQ" />
+
+<PageSection title="How can Pharos help my development?" subSectionLevel={2}>
+  <p>
+    Pharos contains visual and behavioral components that are consistent with our design decisions
+    out of the box. Pharos also seeks to overcome the challenges Bonsai faces around framework
+    interoperability, code bloat, and interdependent components.
+  </p>
```
Comment (daneah) [inline packages/pharos-site/src/pages/faqs.mdx:10]: This bit isn't contextualized enough for an external audience and I don't think it makes sense to keep it here:

```suggestion
  <p>
    Pharos contains visual and behavioral components that are consistent with our design decisions
    out of the box.
  </p>
```

File: packages/pharos-site/src/pages/faqs.mdx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    You can find the status of each component{' '}
+    <PharosLink href="https://pharos.jstor.org/components/component-status">here</PharosLink>. This
+    page will reflect the progress on various aspects of a component's readiness and help you
+    understand what to expect before adopting a component so you can plan accordingly.
+  </p>
+</PageSection>
+
+<PageSection title="What do I do if I still run into issues?" subSectionLevel={2}>
+  <p>
+    Give us the gift of feedback! When you run into issues it is important that you be vocal about
```
Comment (daneah) [inline packages/pharos-site/src/pages/faqs.mdx:27]:
```suggestion
    Give us the gift of feedback! When you run into issues it's important that you be vocal about
```

File: packages/pharos-site/src/pages/faqs.mdx [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    it and not let that inhibit or discourage you from adopting them. We need your help to identify
+    the gaps and sincerely welcome feedback no matter how tough it is.
+  </p>
+  <p>
+    The recommended way to provide feedback is via Github issues. Creating an issue in the
+    repository will provide a common place to track and discuss the nature of the issue, as well as
+    potential solutions.
+  </p>
```
Comment (daneah) [inline packages/pharos-site/src/pages/faqs.mdx:35]: We could replace this whole paragraph with a short blurb and link to the contribution guidelines.

File: packages/pharos-site/src/pages/faqs.mdx [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+  <p>
+    Design systems help bring consistency to platforms, so falling behind may result in duplication
+    of effort in replicating styles and behaviors. It may also result in more catch up. Finally, if
+    you're on an unsupported version of Pharos you may not benefit from the latest bug and security
+    fixes.
+  </p>
+  <p>
+    We'll be managing this by developing a support policy, using semantic versioning, and
+    communicating changes through change logs and regular write ups in Slack.
+  </p>
```
Comment (daneah) [inline packages/pharos-site/src/pages/faqs.mdx:51]: To broaden for an external audience:

```suggestion
  <p>
    We'll be managing this by developing a support policy, using semantic versioning, and
    communicating changes through change logs and discussions.
  </p>
```

File: packages/pharos-site/src/pages/faqs.mdx [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+  title="I adopted a component on my pages just fine, but QA is having trouble testing it. What's the challenge there?"
+  subSectionLevel={2}
+>
+  <p>
+    Pharos components use the shadowDOM which introduces some complexities to testing. To simplify
+    these, we've created Pharos Gems to help other QA teams effectively test Pharos components. It's
+    advised that these Gems be used in QA tests to ensure that the correct elements are being
+    tested.
+  </p>
+</PageSection>
```
Comment (daneah) [inline packages/pharos-site/src/pages/faqs.mdx:64]: This should be removed as it's specific to our internal organization:

```suggestion
```

File: packages/pharos-site/src/pages/faqs.mdx [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+<PageSection title="What do I do if I still run into issues?" subSectionLevel={2}>
+  <p>
+    Give us the gift of feedback! When you run into issues it's important that you be vocal about it
+    and not let that inhibit or discourage you from adopting Pharos. We need your help to identify
+    the gaps and sincerely welcome feedback no matter how tough it is.
+  </p>
+  <p>
+    You can start providing feed back via{' '}
+    <PharosLink href="https://github.com/ithaka/pharos/issues">Github issues</PharosLink>. For more
```
Comment (daneah) [inline packages/pharos-site/src/pages/faqs.mdx:34]:
```suggestion
    <PharosLink href="https://github.com/ithaka/pharos/issues">GitHub issues</PharosLink>. For more
```

File: packages/pharos-site/src/pages/faqs.mdx [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+    on dealing with issues, check out the{' '}
+    <PharosLink href="https://pharos.jstor.org/contributing/development">
+      contribution guideline
```
Comment (daneah) [inline packages/pharos-site/src/pages/faqs.mdx:37]:
```suggestion
      contribution guidelines
```

Or "contribution guide" would work, too.

**Review by daneah — APPROVED** [review]

**Pull request merged**